release.Rmd

```diff
@@ -26,21 +63,304 @@ To get your package ready to release, follow these steps:
+
+This really boils down to which component of the version number is going to be incremented.
+In our workflow, this planned version number is recorded in the GitHub issue that holds the release checklist, but we don't actually increment the version in `DESCRIPTION` until later in the process.
+
+## Initial CRAN release: Special considerations
+
+Every new package receives a higher level of scrutiny from CRAN.
+In addition to the usual automated checks, new packages are also reviewed by a human, which inevitably introduces a certain amount of subjectivity and randomness.
+There are many packages on CRAN that would not be accepted in their current form, if submitted today as a completely new package.
+This isn't meant to discourage you.
+But you should be aware that, just because you see some practice in an established package (or even in base R), that doesn't mean you can do the same in your new package.
+There's a reason the community maintains lists of common "gotchas" for new packages; we have all learned the hard way.
+
+```{r}
+#| eval: false
+usethis:::release_checklist("0.0.1", on_cran = FALSE)
+```
+
+If your package is not yet on CRAN, the checklist begins with a special section that looks something like this:
+
+First release
+
+-   [ ] `usethis::use_cran_comments()`
+-   [ ] Update (aspirational) install instructions in `README`
+-   [ ] Proofread `Title:` and `Description:`
+-   [ ] Check that all exported functions have `@returns` and `@examples`
+-   [ ] Check that `Authors@R:` includes a copyright holder (role 'cph')
+-   [ ] Check licensing of included files
+-   [ ] Review <https://github.com/DavisVaughan/extrachecks>
+
+`usethis::use_cran_comments()` initiates a file to hold submission comments for your package.
+It's very barebones at first, e.g.:
+
+    ## R CMD check results
+
+    0 errors | 0 warnings | 1 note
+
+    * This is a new release.
+
+In subsequent releases, this file becomes less pointless; for example, it is where we report the results of reverse dependency checks.
+This is not a place to wax on with long explanations about your submission.
+In general, you should eliminate the need for such explanations, especially for an initial submission.
+
+We highly recommend that your package have a README file (@sec-readme).
+If it does, this is a good time to check the installation instructions provided there.
+You may need to switch from instructions to install it from GitHub, in favor of installing from CRAN, in anticipation of your package's acceptance.
+
+The `Title` and `Description` fields of `DESCRIPTION` are real hotspots for nitpicking during CRAN's human review.
+Carefully review the advice given in @sec-description-title-and-description.
+Also check that `Authors@R` includes a copyright holder, indicated by the 'cph' role.
+The two most common scenarios are that you add 'cph' to your other roles (probably 'cre' and 'aut') or that you add your employer to `Authors@R:` with the 'cph' and, perhaps, 'fnd' role.
```

Comment:
(The main reason to add `fnd` is that they will then also appear in the footer of your pkgdown website)

Reply:
added